<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSlider" name="verticalSliderKitchen">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="verticalSliderLiving">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="verticalSliderOffice">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="lcdNumberKitchen"/>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumberLiving"/>
    </item>
    <item row="1" column="2">
     <widget class="QLCDNumber" name="lcdNumbeOffice"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kitchen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Living</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Office</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSlider" name="horizontalSliderAussen">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdNumberAussen"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderKitchen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberKitchen</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderOffice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumbeOffice</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderLiving</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberLiving</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAussen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberAussen</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
